arm-network/2017-03-01/swagger/azureWAN.json

```diff
@@ -0,0 +1,3345 @@
+                        "$ref": "#/parameters/GatewayParameter"
+                    },
+                    {
+                        "$ref": "#/parameters/ConnectionNameParameter"
+                    },
+                    {
+                        "$ref": "#/parameters/ApiVersionParameter"
+                    },
+                    {
+                        "name": "parameters",
+                        "in": "body",
+                        "required": true,
+                        "schema": {
+                            "$ref": "#/definitions/ScalableVpnGatewayPutOrPatchParameters"
+                        },
+                        "description": "Parameters supplied to update a vpn connection."
+                    }
+                ],
+                "responses": {
+                    "200": {
+                        "description": "Request successful. Returns the details of the updated gateway.",
+                        "schema": {
+                            "$ref": "#/definitions/VpnConnection"
+                        }
+                    },
+                    "202": {
+                        "description": "Request received successfully. Vpn connection is not yet updated; follow the Location header to poll for final outcome."
+                    },
+                    "400": {
+                        "description": "Bad Request. Failed to validate."
+                    }
+                }
+            },
+            "delete": {
+                "operationId": "AzureVpnConnectionToScalableVpnGateway_Delete",
+                "description": "Deletes a vpn connection.",
+                "parameters": [
+                    {
+                        "$ref": "#/parameters/SubscriptionIdParameter"
+                    },
+                    {
+                        "$ref": "#/parameters/ResourceGroupParameter"
+                    },
+                    {
+                        "$ref": "#/parameters/HubParameter"
+                    },
+                    {
+                        "$ref": "#/parameters/GatewayParameter"
+                    },
+                    {
+                        "$ref": "#/parameters/ConnectionNameParameter"
+                    },
+                    {
+                        "$ref": "#/parameters/ApiVersionParameter"
+                    }
+                ],
+                "responses": {
+                    "200": {
+                        "description": "Request successful. Vpn connection deleted."
+                    },
+                    "202": {
+                        "description": "Request received successfully. Vpn connection deletion is in progress; follow the Location header to poll for final outcome."
+                    },
+                    "204": {
+                        "description": "No connections by the name exist."
+                    },
+                    "400": {
+                        "description": "Cannot delete."
```

Comment:
In order to be consistent with all other Networking swaggers, removing 400 from delete responses. Believe, the default response will be returned as a result.